packages/website/src/components/Uploader/Uploader.js

```diff
@@ -47,15 +47,37 @@ const LogInLink = () => {
   );
 };
 
+const SubscribeLink = () => {
+  const createAccountsUrl = useAccountsUrl();
+
+  return (
+    <Link
+      href={createAccountsUrl("payments")}
+      className="uppercase underline-primary hover:text-primary transition-colors duration-200"
+    >
+      available plans
+    </Link>
+  );
+};
+
 const Uploader = () => {
   const [mode, setMode] = React.useState("file");
   const [uploads, setUploads] = React.useState([]);
 
   const { data: accounts } = useAccounts();
-  const showAccountFeatures =
-    accounts && accounts.enabled !== false && !accounts?.auth_required && !accounts?.authenticated;
-  const disabledComponent =
-    accounts && accounts.enabled !== false && accounts?.auth_required && !accounts?.authenticated;
+
+  // variables extracted from useAccounts response
+  const isAccountsEnabled = accounts?.enabled;
```

Comment:
do we need to add a check to see whether accounts is undefined and handle that?

Reply:
if it's undefined then it's falsy and should serve as false here correctly